scripts/process-csv-images

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# This script processes a given cluster service version file (csv) to replace all occurrences of internal image registries with a deloeran version and generate an image_mirror_mapping file.
+#
+# Example: Running the script and checking the changes produced
+#
+#$ ./process-image-manifests registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-2
```

Comment:
Should this line be this script instead?

Reply:
:roll_eyes: Yes